src/assets/scss/ui/forms/_button.scss

```
@@ -27,3 +27,10 @@ button {
     padding-right: 10px !important;
   }
 }
+.button-primary {
```

I guess we already discussed this to maintain it like following

```suggestion
.button{
...
&.primary{
}
&.secondary{
}
}
```

So you won't have redundant code.